src/app/services/mail/mail.service.ts

```diff
@@ -64,6 +64,21 @@ const DEFAULT_RETRY_PARAMS: MailServiceParams['retryParams'] = {
   minTimeout: 5000,
 }
 
+// TODO #130 Delete references to US SES when SES migration is over (opengovsg/formsg-private#130)
+const START_TS = new Date(config.nodemailer_sg_warmup_start_date).getTime()
+const WARM_UP_DURATION = 6 * 7 * 24 * 60 * 60 * 1000 // 6 weeks
+
+const getWarmUpThreshold = () => {
+  // if START_TS is an empty string, set threshold to 0
+  if (isNaN(START_TS)) {
+    return 0
+  }
```

Comment:
Just for reference: this safety check is not just to catch empty strings, but to catch any input error in our config. Any config input (accidentally) not parsable as an ISO Date would be caught here. 

This should never happen obviously (with config update pairing and all that), but it's just simple defensive programming that can catch "simple" mistakes:

For example:
```javascript
> (new Date(' 2022-11-15T02:47:30')).getTime() // accidental leading space
NaN

> (new Date('2022-11-15T02:47:30 ')).getTime() // accidental trailing space
NaN

> (new Date('2022-11-15T02:47:30+08')).getTime() // invalid timezone suffix definition
NaN
```